### ISSUE TITLE
Problem with replacement of JavaScript String array

### ISSUE THREAD

**Issue by huxi** [thymeleaf/thymeleaf#187]
We have the following code in a `<script>` tag of one of our templates:

```
var userRoles = /*[[${userRoles}]]*/ ['USER','ADMINISTRATOR'];
```

This is transformed into

```
var userRoles=['ROLE1','ROLE2'], 'ADMINISTRATOR'];
```

, i.e. only the first part of the array prototype is removed.

This seems to have been caused by the update from 2.0.16 to 2.0.17.

**Comment by huxi** [comment]
Sorry, missed #168.

**Comment by danielfernandez** [comment]
Actually, your specific scenario wasn't completely solved in 2.0.18 yet. It is now, thanks for reporting.

I've added a test with your example code, which now runs OK: https://github.com/thymeleaf/thymeleaf-tests/blob/20130711/src/test-common/resources/engine/attrprocessors/inline/inline06.thtest

**Comment by huxi** [comment]
Oops, awesome. Thanks! :D
